gtk4/src/auto/tree_store.rs

```diff
@@ -60,7 +60,7 @@ pub trait TreeStoreExt: 'static {
 
     fn prepend(&self, parent: Option<&TreeIter>) -> TreeIter;
 
-    fn remove(&self, iter: &TreeIter) -> bool;
+    //fn remove(&self, iter: /*Unimplemented*/TreeIter) -> bool;
```

Comment:
Hm why? `TreeIter` bindings exist

Reply:
Hmm no idea, maybe the annotation is broken on the latest release ? I will check

Reply:
Something seems clearly broken, yes. Two lines above the other function works fine :)

Reply:
Ah it uses inout
```xml
      <method name="remove" c:identifier="gtk_tree_store_remove">
        <doc xml:space="preserve">Removes @iter from @tree_store.  After being removed, @iter is set to the
next valid row at that level, or invalidated if it previously pointed to the
last one.</doc>
        
        <return-value transfer-ownership="none">
          <doc xml:space="preserve">%TRUE if @iter is still valid, %FALSE if not.</doc>
          <type name="gboolean" c:type="gboolean"/>
        </return-value>
        <parameters>
          <instance-parameter name="tree_store" transfer-ownership="none">
            <doc xml:space="preserve">A #GtkTreeStore</doc>
            <type name="TreeStore" c:type="GtkTreeStore*"/>
          </instance-parameter>
          <parameter name="iter" direction="inout" caller-allocates="0" transfer-ownership="full">
            <doc xml:space="preserve">A valid #GtkTreeIter</doc>
            <type name="TreeIter" c:type="GtkTreeIter*"/>
          </parameter>
        </parameters>
      </method>
```

Reply:
Needs to be manually implemented for now then

Reply:
I've opened https://github.com/gtk-rs/gtk4-rs/issues/40 to keep track of that